### PR TITLE
V2.10.0

### PR DESCRIPTION
**Pull request by XiaoningLiu**
2018.06 Version 2.10.0

ALL
* Updated storage service version to 2018-03-28.

BLOB
* Fixed a bug that `DeleteRetentionPolicy.Days` should be `number` instead of `string` when calling `getServiceProperties`.
* Added a method `getAccountProperties` to `blobService`.
* Added a method `createBlockFromURL` to `blobService`.
* Added support for static website service properties (in preview).

### CODE REVIEW

File: browser/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 Note: This is the change log file for Azure Storage JavaScript Client Library.
 
+2018.06 Version 2.10.100
```
Comment (jiacfan) [inline browser/ChangeLog.md]: what's the min version 100 stands for?

File: lib/azure-storage.js [hunk 2]
```diff
@@ -314,6 +314,33 @@ exports.generateAccountSharedAccessSignature = function(storageAccountOrConnecti
 * @property   {CorsRule[]}          Cors.CorsRules         Groups settings for a `[CORS rule]{@link CorsRule}`.
 */
 
+/**
+* The properties of a storage account.
+* @typedef    {object}              AccountProperties
+* @property   {string}              SkuName                The header that specifies storage SKU, also known as account type.
+* @property   {string}              AccountKind            The header that describes the flavour of the storage account, also known as account kind.
+* The properties of a blob storage service, including properties of Storage Analytics, CORS (Cross-Origin Resource Sharing) rules and Static Webiste configurations.
```
Comment (jiacfan) [inline lib/azure-storage.js]: This format seems strange. Please double check if this is as expected.

File: lib/services/blob/blobservice.core.js [hunk 3]
```diff
@@ -2116,7 +2171,14 @@ BlobService.prototype.createBlobSnapshot = function (container, blob, optionsOrC
 };
 
 /**
-* Starts to copy a blob to a destination within the storage account. The Copy Blob operation copies the entire committed blob.
+* Starts to copy a blob or an Azure Storage file to a destination blob.
+*
+* For an asynchronous copy(by default), this operation returns a object including a copy ID which
```
Comment (jiacfan) [inline lib/services/blob/blobservice.core.js]: Seems need optimize, currently always asynchronous?

File: browser/ChangeLog.md (hunk 1 above)
Comment (XiaoningLiu) [reply browser/ChangeLog.md]: To differ from the Node.js version number. Every JS new minor version starts with patch version 100.

File: lib/azure-storage.js (hunk 2 above)
Comment (XiaoningLiu) [reply lib/azure-storage.js]: Thanks. Should be issues introduced by branch merging. Will update here.

File: lib/azure-storage.js [hunk 4]
```diff
@@ -314,6 +314,33 @@ exports.generateAccountSharedAccessSignature = function(storageAccountOrConnecti
...
+*/
+
+/**
```
[the remark this answers is not in the document]
Comment (XiaoningLiu) [reply lib/azure-storage.js]: Thanks. Should be issues introduced by branch merging. Will update here.

File: lib/services/blob/blobservice.core.js (hunk 3 above)
Comment (XiaoningLiu) [reply lib/services/blob/blobservice.core.js]: Always asynchronous.

**Pull request merged**